src/Core/Services/CryptoService.cs

```diff
@@ -29,10 +30,12 @@ public class CryptoService : ICryptoService
 
         public CryptoService(
             IStateService stateService,
-            ICryptoFunctionService cryptoFunctionService)
+            ICryptoFunctionService cryptoFunctionService,
+            ILogger logger)
```

Comment:
⚠️ Is this compiling?  I don't see the change on `iOSCoreHelpers` and `MainApplication` to pass the `logger` instance to the initialization of the `CryptoService`

Reply:
Fixed in commit [fc8d058](https://github.com/bitwarden/mobile/pull/2893/commits/fc8d058046e23004a7c062c1f384984b959a9547)